GitExtensions/GitExtensions.csproj

```diff
@@ -132,4 +132,12 @@
     </ItemGroup>
     <Copy SourceFiles="@(Translations)" DestinationFolder="$(TargetDir)Translation" ContinueOnError="false" />
   </Target>
+  <Target Name="BuildEasyHookDll" AfterTargets="AfterBuild">
```

Comment:
@NikolayXHD Minor build issue: I can't build the solution on my machine. As a C# dev, I hope that I don't have to install MS VC++ (some 4.5GB) to build this project. Is that possible?
 ``` 
EasyHookDLL.vcxproj:L20:  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
```

Reply:
Made it conditional, tell me if it is was enough to fix the build in your machine.

Reply:
It did, I can build successfully now. Thank you very much.

Reply:
We have several C++ projects in the solution, so to build an installer one needs to have C++ installed.
However I don't feel it is necessary to force people to build and use EasyHook locally, if we can avoid it and developers don't desire to do so.

Reply:
I was doing this PR with graceful degradation in mind. There are some changes yet to be made with respect to it, but it's not hard and I was going to do them anyway.